### ISSUE TITLE
Should display the properties defined in behaviors.

### ISSUE THREAD

**Issue by blasten** [PolymerElements/iron-component-page#39]
The main doc should display the properties defined in behaviors, so people won't need to click on each behavior to know what properties they can use.

**Comment by ronnyroeller** [comment]
+1

**Comment by notwaldorf** [comment]
Note for whoever decides to tackle this: It's confusing that somehow this is fixed in `polymer-element-catalog` but not in `iron-component-page` 😓

**Comment by miztroh** [comment]
It would also be nice if this worked with behaviors pulled in from other bower packages.  For example, I have package A that pulls in a behavior from package B and a behavior from package C.  It would be nice if the package A docs showed all the properties and methods inherited from the B / C behaviors.

**Comment by jahglow** [comment]
@miztroh Absolutely: For instance if my element that I'm documneting has a behavior in the same folder and an inherited behavior from another element, it won't show any docs from any behaviors because it can't find that even though that behavior is installed and properly linked from the bower-components.

**Comment by garlicnation** [comment]
Will be fixed by #86

**Comment by venicia** [comment]
+1
